rockPaperScissors.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
```

Comment:
Again, you should include comments because I'm not sure why you did some of the things you did. You again had warnings; make sure these are fixed before submission.